src/Voron/Data/Tables/Table.cs

```diff
@@ -1618,6 +1618,37 @@ public TableValueHolder ReadLast(TableSchema.FixedSizeKeyIndexDef index)
             }
         }
 
+        public IEnumerable<SeekResult> SeekBackwardFromLast(TableSchema.AbstractTreeIndexDef index, long skip = 0)
```

Comment:
@arekpalinski can you review this?

Reply:
We already have a method that we can use instead of introducing a new one I think:


https://github.com/ravendb/ravendb/blob/v6.0/src/Voron/Data/Tables/Table.cs#L1247-L1283



just pass `prefix: null`  and then don't call `it.SetRequiredPrefix(prefix);` if that's the case

Reply:
Done